AccountData/src/main/java/skedgo/tripkit/account/data/UserKeyRepositoryImpl.kt

```diff
@@ -0,0 +1,25 @@
+package skedgo.tripkit.account.data
+
+import android.content.SharedPreferences
+import rx.Completable
+import rx.Single
+import skedgo.tripkit.account.domain.UserRepository
+
+internal class UserKeyRepositoryImpl(
+    private val prefs: SharedPreferences
+) : UserRepository {
+
+  private val userKeyKey = "userKey"
+
+  override fun getUserKey(): Single<String> =
+      Single.fromCallable {
+        prefs.getString(userKeyKey, "")
+      }
+
+  override fun setUserKey(userKey: String): Completable =
+      Completable.fromCallable {
+        prefs.edit()
+            .putString(userKey, userKey)
+            .apply()
+      }.doOnEach {  }
```

Comment:
Let's not allow setting the UUID? @elagarrigue 
if the key exists, then use it. Otherwise, generate a new one then store and return it.

Reply:
So the idea is the merge `GenerateUserKey` into `UserKeyRepositoryImpl`

Reply:
Ok, I'll move it